astropy/io/misc/tests/test_hdf5.py

```diff
@@ -51,23 +49,19 @@ def test_write_nopath(tmpdir):
     test_file = str(tmpdir.join('test.hdf5'))
     t1 = Table()
     t1.add_column(Column(name='a', data=[1, 2, 3]))
-    with pytest.raises(ValueError) as exc:
+    with pytest.warns(UserWarning, match="table path was not set via the path= argument"):
```

Comment:
This test was never run before because of name clash with the one on L57, hence the dramatic looking change.

Reply:
They do look similar though. Should I just delete this one?